libs/platform/user/ebpf_leak_detector.cpp

```diff
@@ -54,6 +54,7 @@ _ebpf_leak_detector::dump_leaks()
         }
         std::cout << std::endl;
     }
+    ebpf_assert(_allocations.empty());
```

Comment:
This can be removed as it's now being done just below (pulled in from a different PR that got merged).